ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1487,7 +1487,8 @@ const RuntimePanel = fp.flow(
                           runtimeCtx = {runtimeCtx}
                   />
               </FlexRow>
-              {currentRuntime && currentRuntime.errors && <ErrorMessage iconPosition={'top'} iconSize={16}>
+              {currentRuntime && currentRuntime.errors && currentRuntime.errors.length > 0
+                && <ErrorMessage iconPosition={'top'} iconSize={16}>
```

Comment:
Could you also update the default runtime model to have `errors: []`? This will more accurately reflect a Running runtime model:

https://github.com/all-of-us/workbench/blob/master/ui/src/testing/stubs/runtime-api-stub.ts#L27

Reply:
Yep can do